controllers/ovncontroller_controller.go

```diff
@@ -553,6 +553,37 @@ func (r *OVNControllerReconciler) reconcileNormal(ctx context.Context, instance
 	instance.Status.DesiredNumberScheduled = dset.GetDaemonSet().Status.DesiredNumberScheduled
 	instance.Status.NumberReady = dset.GetDaemonSet().Status.NumberReady
 
+	// Define a new DaemonSet object for the OVS (ovsdbserver+ovsvswitchd)
+	ovsDaemonSet, err := ovncontroller.CreateOVSDaemonSet(instance, inputHash, serviceLabels, serviceAnnotations)
+	if err != nil {
+		Log.Error(err, "Failed to create OVS DaemonSet")
+		return ctrl.Result{}, err
+	}
+	dset = daemonset.NewDaemonSet(
```

Comment:
maybe You could use some different variable here, like e.g. "ovsdset" or something like that, wdyt?

Reply:
Of course, it will improve the readability of the code and there is no need to save variables.